styles/styles.css

```
@@ -30,6 +38,55 @@ body {
 .page-header {
   background-color: #ffffff;
   color: #000000;
+  width: 1200px;
+  margin: 0 auto;
+}
+
+.inner-header {
+  background-image: url("../images/catalog-background.jpg");
+  background-repeat: no-repeat;
+  color: #ffffff;
+  background-color: #0d046e;
+  background-size: cover;
+  background-position: center;
+  width: 1200px;
+  margin: 0 auto;
+  display: flex;
+
+  flex-wrap: wrap;
+}
+
+.navigation {
+  display: flex;
+  width: 1060px;
+  margin: 0 auto;
+}
+
+.navigation img {
+  width: 139px;
+  margin-right: 30px;
+}
+
+.navigation-list {
+  display: flex;
+  justify-content: space-between;
+  width: 450px;
+  margin: 0;
+  padding: 0;
+  margin-right: 177px;
+}
+
+.navigation-user {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: auto;
+  margin: 0;
```

Тебе нужно либо их местами поменять, либо маржин лефт вообще убрать
```suggestion
  margin: 0;
  margin-left: auto;
```

Потому что то что снизу css, перезаписывает то что по каскаду выше. А значит ты обнуляешь маржин лефт который тольок что задал